### PR TITLE
[MM-27954] Fix JS error on sys console groups page

### PR DESCRIPTION
**Pull request by fmunshi**
#### Summary
- Fixes an issue where `this.setState` undefined error shows up, prevent this error by making closeFilters an arrow function  

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-27954

image

### CODE REVIEW

File: components/admin_console/group_settings/group_settings.jsx [hunk 1]
```diff
@@ -14,7 +14,10 @@ import {getSiteURL} from 'utils/url';
 
 export default class GroupSettings extends React.PureComponent {
     static propTypes = {
-        isDisabled: PropTypes.func,
+        isDisabled: PropTypes.oneOfType([
+            PropTypes.func,
+            PropTypes.bool,
+        ]),
```
Comment (sbishel) [inline components/admin_console/group_settings/group_settings.jsx:20]: Is this related or something separate?

**Review by sbishel — APPROVED** [review]

File: components/admin_console/group_settings/group_settings.jsx (hunk 1 above)
Comment (fmunshi) [reply components/admin_console/group_settings/group_settings.jsx:20]: This was a separate issue - isDisabled can be both a function or a bool... there was a console error that showed up on this page because of that

**Review by calebroseland — APPROVED** [review]

**Review by srkgupta — APPROVED** [review]
Tested and the issue is fixed now and is working fine and there are no JS errors thrown. Approving the PR
